src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -757,7 +757,9 @@ static unique_ptr<Scene> _sceneBuildHelper(const SvgNode* node, const Box& vBox,
 static void _applySvgViewFlag(const Scene* scene, float& vx, float& vy, float& vw, float& vh, float& w, float& h, SvgViewFlag viewFlag)
 {
     if (!((uint32_t)viewFlag & (uint32_t)SvgViewFlag::Viewbox)) {
-        scene->bounds(nullptr, nullptr, &vw, &vh, false);
+        scene->bounds(&vx, &vy, &vw, &vh, false);
+        vw += vx;
```

Comment:
removing these 4 lines:
```
        vw += vx;
        vh += vy;
        vx = 0.0f;
        vy = 0.0f;
```
ensures that whole images without viewbox and width/height info are drawn

Reply:
I already pressed approve, but please check again
I tested this pr using godot [test image.](https://www.kenney.nl/assets/board-game-icons)
As tested on this page, the images didn't look aligned. (https://github.com/thorvg/thorvg/issues/1332)
![image](https://user-images.githubusercontent.com/7413838/229439330-34ee48cb-31c1-40ba-b7d9-0a6a08ea2fa8.png)
I don't know what i missed.

Anyway, if my understanding is correct
After applying this patch, all svg images with no viewbox defined will be aligned to 0,0.
In ubuntu-imageviewer, chrome, or firefox, the results don't sorted by 0,0.
Is there an svg viewer example that shows this sorted by 0,0? (Not an image editing tool..)

(I don't think there is a standard(?) definition for this.(I cant find it) so.,.  maybe we can support this.)

Reply:
@JSUYA as I mentioned in #1332, I forced x=y=0. but I'm not sure what should be drawn in such cases. and I agree that since we are not sure, and godot need whole images, lets solve it. If someone will point out that this is an error, we'll take care of that